__init__.py

```diff
@@ -116,7 +151,7 @@ def upgrade_possible(self):
         else:
             return False
 
-    @intent_handler(IntentBuilder("").require("Check").require("Version"))
+    @intent_handler(AdaptIntent("").require("Check").require("Version"))
```

Comment:
One of the nice things about using the AdaptIntent is that there is no longer a need to pass an empty string to the constructor.